<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waveshare Config Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>641</width>
      <height>321</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Received / Sent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>77</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This only works with 1 device connected</string>
    </property>
    <property name="text">
     <string>Read Device Address</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Software Version</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>246</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Scan for Devices</string>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Com Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Configuration">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
